portfolio/src/main/webapp/loginCheck.js

```diff
@@ -0,0 +1,25 @@
+document.addEventListener("DOMContentLoaded", displayPage, false);
+
+function displayPage(e){
+  fetch('/loginStatus').then(response => response.json()).then(data =>{
+    if(data.loginStatus == false){
+      const body = document.getElementById("content");
+      body.innerHTML = "";
+
+      const error = document.createElement("h1");
+      error.appendChild(document.createTextNode("Error: Not logged in!"));
+      body.appendChild(error);
+
+      const loginLink = document.createElement("a");
+      loginLink.href = data.url;
+      loginLink.appendChild(document.createTextNode("Click here to log in."));
+      body.appendChild(loginLink);
+    }
+    else{
+      const logoutLink = document.getElementById("logout");
+      const logoutLinkText =document.getElementById("logoutText");
```

Comment:
This is also misnamed. Calling it Text implies that this is a string when it is actually an element that is designed just to hold text.